checker.cs

```diff
@@ -3,16 +3,30 @@
 
 class Checker
 {
-    static bool vitalsAreOk(float bpm, float spo2, float respRate) {
+    static bool bpmIsOk(float bpm){
```

Comment:
the ...IsOk functions are very similar. If they accept the limits also, they can just be one 'vitalIsInRange' function, to be used for any vital with limits.